### DISCUSSION TITLE
Using name_template in 1.1.6

### DISCUSSION THREAD

**Discussion by von-Chaps** [ekutner/home-connect-hass#397] [Community Support] [answered]
I just updated to 1.1.6 and am having trouble using name_template. I used to have this configured in the configuration.yaml, but it seems that is not supported now. The [README ](https://github.com/ekutner/home-connect-hass/tree/1.1.6?tab=readme-ov-file#advanced-options) says;

>### Advanced options
>These options will only show up when the user has enablde "Advanced mode" in the user profile.
> - **Name Template** - Defines the template used for rendering entity names. 

However, I can not discover (or find any docs on) how to enable "Advanced mode". Can you advise please?

**Comment by ekutner** [comment, marked as answer]
It's in your profile settings. Click your user name on the bottom of the left toolbar (under Notifications) then Advanced Mode will be the last option in the "User Settings" section.

**Comment by von-Chaps** [comment]
Perfect, thank you. For some reason I thought this was a Home Connect Alt setting, but I did puzzle as to what "user profile" meant in this context.
